app/test_main.py

```diff
@@ -1,3 +1,65 @@
 from app.main import get_human_age
+import pytest
+from typing import Any
 
-# write your code here
+
+class TestValues:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_value",
+        [
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="Correct values for 15, 15"),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="Correct values for, 24, 24"),
+            pytest.param(
+                28,
+                29,
+                [3, 3],
```

Comment:
The expected value for the test case with inputs `(28, 29)` should be `[3, 2]` according to the conversion rules provided in the task description.